docs/guide/in-depth/code-splitting.md

```diff
@@ -0,0 +1,225 @@
+# Code Splitting
+
+Code splitting is the process of creating chunks from modules. This chapter describes its behavior and the principles behind it.
+
+- Code splitting is not controllable. It runs following certain rules.
+- Thus, we will also refer to it as _`automatic chunking`_ against _`manual chunking`_ done by `advancedChunks`.
+
+## `Entry chunk`s
```

Comment:
I don't think we should wrap "entry chunk" with code backticks - these are more like a special term rather than inline code, so maybe just make them bold, e.g. **entry chunks**

Reply:
Got it.